core/src/test/java/org/jboss/pnc/rex/core/NotificationTest.java

```diff
@@ -89,17 +89,17 @@ public class NotificationTest {
     TransitionRecorder recorder;
 
     @BeforeEach
-    void before() {
+    void before() throws InterruptedException {
+        Thread.sleep(100);
```

Comment:
A running(race) condition, some async jobs were still ongoing after a test completes and after a forceful clear of cache (in AfterEach) they started complaining a lot because their tasks were not present. I've moved the sleep so that the ongoing jobs can fail in the background before affecting other tests.

Reply:
TBF, I can make it prettier. Having a Thread.sleep() in before each is ugly.